SecondSight/AprilTags/Detector.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, yaw, pitch, roll, left_right, up_down, distance, rms, tagid):
         self.distance_std = None
         self.error = None
 
+    # xr,yr,theta_r
+    def makeRelativeToRobot(self, pos: List[float]):
```

Comment:
New methods should also be added as tests. Even if it only does something minor adding a test gives us confidence we won't accidentally break this in some future change

Let me know if you want to work together on adding the tests